### PR TITLE
 Move python files from cloud code generator to content_builder

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

##### SUMMARY

This PR moves the [python scripts](https://github.com/ansible-collections/gouttelette/tree/main/gouttelette/cmd) from [gouttelette](https://github.com/ansible-collections/gouttelette) to ansible.content_builder.

This will close https://issues.redhat.com/browse/ACA-850

A new PR will be pushed to convert these python scripts to filter plugins. [ACA-851 to ACA-854](https://issues.redhat.com/browse/ACA-562) are written to track that part of the work.

Verified using
`ansible-playbook build.yaml -e manifest_file=MANIFEST.yaml`

```
MANIFEST.yaml
---
collection:
  path: /home/github/collections/ansible_collections/amazon/cloud
  namespace: amazon
  name: cloud
plugins:
  - type: module_openapi
    name: "amazon_cloud"
    content: cloud
    module_version: 1.0.0
    api_object_path:  api_spec
    resource: config
    action: generate_all
    unique_key: ""
    rm_swagger_json: ""
    version: "1.0.0"
    author: ""
   
``` 
More details on the parameters to be given in the MANIFEST.yaml are available in [this doc ](https://docs.google.com/document/d/1Y7Kisim-rVAmj4XcHpZUZrPEU9qEHLrfV3zEjU6HCqM/edit?usp=sharing)

##### ISSUE TYPE

- Feature Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by GomathiselviS** [comment]
cc @NilashishC @justjais I have added support for cloud content (openapi based) generation to content_builder. Please review and let me know your comments.

File: roles/module_openapi_cloud/templates/content_library_data.py
Comment (NilashishC) [inline roles/module_openapi_cloud/templates/content_library_data.py:3]: I'm wondering if templates is the correct directory to put this in, given this is a Python file containing a dictionary?

File: roles/module_openapi_cloud/templates/generator.py
Comment (NilashishC) [inline roles/module_openapi_cloud/templates/generator.py:10]: Same as above. Would these get updated once your next PR is merged?

**Review by NilashishC — CHANGES_REQUESTED** [review]
I'm slightly -1 on merging this with the main branch before the scripts are converted to plugins. Assuming that a python3 executable is available leads to multiple issues as we've encountered with `module_openapi_security` and we do not want to merge something on main that _might_ be broken in certain conditions. Also, I would like to have a +1 from @justjais since this refactors `module_openapi_security`. My suggestion is to push this to a new branch in this repo and open subsequent PRs against that until all changes are in place.

File: roles/run/files/schema.json
Comment (NilashishC) [inline roles/run/files/schema.json:113]: Can we also update README to reflect the changes and the new options that are being added?

File: roles/module_openapi_cloud/templates/generator.py
Comment (GomathiselviS) [reply roles/module_openapi_cloud/templates/generator.py:10]: yes, as discussed this file will move into plugin_utils.

**Comment by GomathiselviS** [comment]
duplicate of #43

**Pull request closed without merging**
